graftm/run.py

```diff
@@ -44,7 +45,7 @@ def __init__(self, args):
         self.setattributes(self.args)
 
     def setattributes(self, args):
-        
+        self.t = Taxonomy()
```

Comment:
I recall I was going to use the Taxonomy class elsewhere in the GraftM code (explained in the commit description)